scripts/migrations/populate_points_for_h3.py

```diff
@@ -28,7 +28,7 @@ def main(ctx, db_url, verbosity):
 
 
 def populate_points(session: Session):
-    distinct_h3_select = select(Thing.h3).filter(Thing.h3.isnot(None)).distinct()
+    distinct_h3_select = select(Thing.h3).filter(Thing.h3.isnot(None)).distinct()  # type: ignore
```

Comment:
This is an example of whatever sqlmodel is doing making mypy unhappy.  Luckily they support just turning off type checking on a line by line basis.

Reply:
It is a bit odd - seems clear what the types are there.